lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl

```diff
@@ -160,7 +160,7 @@ status_result(_Status = false) -> connecting.
 %%========================================================================================
 
 do_batch_insert(InstanceId, BatchReqs, InsertPart, Tokens, State) ->
-    SQL = emqx_plugin_libs_rule:proc_batch_sql(BatchReqs, InsertPart, Tokens),
+    SQL = emqx_plugin_libs_rule:proc_batch_mysql(BatchReqs, InsertPart, Tokens),
```

Comment:
Q: since this is also used in TDengine, maybe the function name could be something more generic (perhaps the old name)?

Either this or "fork" the old function so that TDengine uses the old version?

Reply:
Did that as part of https://github.com/emqx/emqx/pull/10095/commits/2544742b0cf02aed1707b068237bb3762651b663. I tried to raise some awareness there about pgsql in a comment, would be glad if you take another look.